src/ledger.ts

```diff
@@ -1518,7 +1447,7 @@ export class LedgerRegisterWalletPolicy extends LedgerBitcoinV2WithRegistrationI
     return messages;
   }
 
-  async run(): Promise<string> {
```

Comment:
Wonder why this was required to be removed.

Reply:
Inheritance conflict.